CONTRIBUTING.rst

```diff
@@ -78,8 +79,8 @@ Ready to contribute? Here's how to set up `git_wrapper` for local development.
 5. When you're done making changes, check that your changes pass flake8 and the
    tests, including testing other Python versions with tox::
 
-    $ flake8 git_wrapper tests
-    $ py.test tests
+    $ make lint
+    $ make tests
```

Comment:
Looks good, with minor change that the make target is 'test' rather than 'tests'